tests/zquantum/core/estimation/estimation_test.py

```diff
@@ -374,13 +539,15 @@ def test_calculate_exact_expectation_values(self, simulator, estimation_tasks):
         )
         assert len(expectation_values_list) == 3
         for expectation_values, task in zip(expectation_values_list, estimation_tasks):
+            if () in task.operator.terms.keys():
+                for i, term in enumerate(task.operator.terms.items()):
+                    if term[0] == ():
+                        assert expectation_values.values[i] == term[1]
```

Comment:
I fixed the `MockQuantumSimulator` since it always stored 1. in the expectation values, causing this test to fail.